docs/development/best_practices/rust.md

```diff
@@ -1,7 +1,7 @@
 # Rust Style Guide
 
 We follow the
-[Rust Style Guide](https://doc.rust-lang.org/1.0.0/style/style/README.html),
+[Rust Style Guide](https://github.com/rust-lang/style-team/blob/master/guide/guide.md),
```

Comment:
@eventualbuddha  should we link to this at a specific commit instead of master so that its versioned? Do we know if our code currently meets this?